snapshots/who/2023-11-01/who_statins.csv.dvc

```diff
@@ -0,0 +1,30 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/origin/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: General availability of statins in the public health sector
+    description: |-
+      This dataset reflects the availability of statins in primary health care facilities within the public health sectors of various countries. A country is deemed to have statins "generally available" if they are present in 50% or more of its pharmacies. Data is sourced primarily from official country responses to the WHO NCD Country Capacity Survey, with key informant surveys as the preferred data collection method. Collection occurs every two years. For further information, contact Leanne Riley at rileyl@who.int.
+    date_published: '2022'
+    version_producer: '2022-04-25'
+    # Citation
+    producer: World Health Organisation
+    citation_full: |-
+      General availability of statins in the public health sector, World Health Organization. Licence: CC BY-NC-SA 3.0 IGO.
+    attribution_short: WHO
+
+    # Files
+    url_main:
+      https://www.who.int/teams/health-product-policy-and-standards/assistive-and-medical-technology/medical-devices/global-atlas-of-medical-devices
```

Comment:
It might be worth using a web archive link here, we often have issues with broken links and the WHO.